### PR TITLE
N30: rafx

### PR DESCRIPTION
**Pull request by aclysma**
Added update for rafx

_Part of #912_

### CODE REVIEW

**Review by ozkriff — APPROVED** [review]

File: content/news/030/index.md [hunk 1]
```diff
@@ -76,6 +76,35 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+[Rafx][rafx] is a multi-backend renderer that optionally integrates with the
+[distill][rafx-distill] asset pipeline.
+
+Since the previous rafx update in this newsletter (6 months ago!), many new
+features have been introduced to improve performance and image quality. The
+main rendering pipeline has also been split into "modern" and "basic"
+pipelines. Unlike the basic pipeline which focusses on wide compatibility,
+the modern pipeline uses forward-clustered lighting and handles hundreds
+of shadow-casting lights. It is targetting compute shaders and will be adding
```
Comment (17cupsofcoffee) [inline content/news/030/index.md:92]:
```suggestion
of shadow-casting lights. It is targeting compute shaders and will be adding
```

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/news/030/index.md (hunk 1 above)
Comment (17cupsofcoffee) [reply content/news/030/index.md:92]: I can't commit to your branch, can you resolve this before I merge?

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
This looks ready to go, but I don't have access to the branch to resolve the conflicts myself.

**Comment by 17cupsofcoffee** [comment]
sneaky manual merge 🥷

**Pull request merged**